src/mixins/java/org/spongepowered/common/mixin/core/world/damagesource/IndirectEntityDamageSourceMixin.java

```diff
@@ -70,8 +80,8 @@ public String toString() {
             .add("Type", resourceKey)
             .add("Source", this.shadow$getDirectEntity())
             .add("IndirectSource", this.shadow$getEntity());
-        if (this.impl$creator != null) {
-            helper.add("SourceOwner", this.impl$creator);
+        if (this.impl$creatorUUID != null) {
+            helper.add("SourceOwner", this.impl$creatorUUID);
```

Comment:
SourceCreator...need to remove this flip flop of Owner/Creator in this codebase eventually...